spec/services/proofing/resolution/progressive_proofer_spec.rb

```diff
@@ -95,363 +131,387 @@
           expect(device_profiling_result.review_status).to eq('pass')
         end
       end
-    end
 
-    context 'ThreatMetrix is disabled' do
-      before do
-        allow(FeatureManagement).to receive(:proofing_device_profiling_collecting_enabled?).
-          and_return(false)
+      context 'LexisNexis Instant Verify A/B test enabled' do
+        let(:residential_instant_verify_proof) do
+          instance_double(Proofing::Resolution::Result)
+        end
+        let(:instant_verify_workflow) { 'equitable_workflow' }
+        let(:ab_test_variables) do
+          {
+            ab_testing_enabled: true,
+            use_alternate_workflow: true,
+            instant_verify_workflow: instant_verify_workflow,
+          }
+        end
+
+        before do
+          allow(instant_verify_proofer).to receive(:proof).
+            and_return(residential_instant_verify_proof)
+          allow(residential_instant_verify_proof).to receive(:success?).and_return(true)
+        end
 
-        allow(instance).to receive(:proof_id_address_with_lexis_nexis_if_needed).
-          and_return(resolution_result)
-        allow(resolution_result).to receive(:success?).and_return(true)
-        allow(instant_verify_proofer).to receive(:proof)
+        it 'uses the selected workflow' do
+          lniv = Idv::LexisNexisInstantVerify.new(dcs_uuid)
+          expect(lniv).to receive(:workflow_ab_testing_variables).
+            and_return(ab_test_variables)
+          expect(Idv::LexisNexisInstantVerify).to receive(:new).
+            and_return(lniv)
+          expect(Proofing::LexisNexis::InstantVerify::Proofer).to receive(:new).
+            with(hash_including(instant_verify_workflow: instant_verify_workflow)).
+            and_return(instant_verify_proofer)
+
+          proof
+        end
       end
-      it 'returns a disabled result' do
-        result = subject
 
-        device_profiling_result = result.device_profiling_result
+      context 'remote flow does not augment pii' do
```

Comment:
this is going to be a great spec to have to ensure we're not impacting the remote flow. thank you for adding this!